README.md

```diff
@@ -53,7 +53,7 @@ Repository | Description
 [YARA](https://github.com/InQuest/awesome-yara)                                     | List of awesome YARA rules, tools, and people
 [Hacker Roadmap](https://github.com/sundowndev/hacker-roadmap)                                     | A guide for amateur pen testers and a collection of hacking tools, resources and references to practice ethical hacking.
 
-## Other useful repositories
```

Comment:
55

Reply:
Ok move on

Reply:
Muito bem...aprovado...eu tanbem quero

Reply:
@sri_srillu 
Hack this insta gram account